src/oidc.rs

```diff
@@ -4,9 +4,26 @@ use std::convert::TryInto;
 use tame_oauth::Error;
 use url::form_urlencoded::Serializer;
 
+/// Request object sent in a token exchange request
+#[derive(serde::Deserialize, serde::Serialize, Debug)]
+pub struct TokenExchangeRequest {
```

Comment:
I'm not sure this is used anywhere

Reply:
No it's not :D Left-over from the previous installment where I wanted to pass in just a struct instead of args. Nicely spotted.